index.md

```diff
@@ -1,61 +1,19 @@
 ---
 layout:     page
-title:      The Literature Review site of VITAL
+title:      The literature repo of the VITALab
```

Comment:
I think "repository" should not be in the title of the site, since it is more a mean than an end.

I would suggest keeping "review", which better conveys our goal. "archive" would be another option.

References:
* https://en.wikipedia.org/wiki/Review_article
* https://en.wiktionary.org/wiki/repository
* https://en.wiktionary.org/wiki/archive#English